cls/log/cls_log.cc: reduce logging noise

wjwithagen opened the pull request:
- The other reference in the source as already at 20.
      ./src/cls/timeindex/cls_timeindex.cc:85:
	CLS_LOG(20, "storing entry at %s", index.c_str());

   And we need not always know where in the log items are stored.
   So it looks like a leftover debug feature.

Fixes: http://tracker.ceph.com/issues/19835
Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

jrosenboom commented:
Thanks for this patch, please also mark for backporting.

cbodley commented:
@wjwithagen created a backport ticket at http://tracker.ceph.com/issues/19835, could you please add to your commit message?

wjwithagen commented:
@cbodley 
Yup, added commit message, but the wrong way around, I guess. Added it to the tracker.
But will also add tracker-id to the actual commit.
